### DISCUSSION TITLE
show board

### DISCUSSION THREAD

**Discussion by malaschitz** [fairy-stockfish/Fairy-Stockfish#551] [General]
In some variations it is not easy to find out what is the correct notation for the moves. For example, for castling in Chess960 or Seirawan. The setposition command, with a list of moves, ignores incorrect moves but does not write which moves are incorrect. It would help if there was a command that shows the position (fen) after the `position startpos moves ...` command

**Comment by ianfab** [comment]
There is, you can use `d`.

And if you want to get the list of legal moves you can use `go perft 1`.

**Comment by malaschitz** [comment]
Thank You very much !
